library/src/main/java/com/cube/storm/content/lib/manager/DefaultUpdateManager.java

```diff
@@ -0,0 +1,518 @@
+package com.cube.storm.content.lib.manager;
```

Comment:
95% of the code here isn't new and is from `UpdateManager`. The only new code is to implement the new child class interfaces.